### PR TITLE
Gather all audit logs, not only the latest one

### PR DESCRIPTION
**Pull request by soltysh**
Audit logs are rotated when server restarts, thus we need to gather all logs from audit directory, not just the latest one. I'm also adding gathering openshift api audit logs.

/assign @mfojtik @smarterclayton @bbguimaraes

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 1]
```diff
@@ -447,7 +447,10 @@ objects:
           done < /tmp/containers
 
           echo "Gathering kube-apiserver audit.log ..."
-          FILTER=gzip queue /tmp/artifacts/nodes/masters-kube-apiserver-audit.log.gz oc adm node-logs --role=master --path=kube-apiserver/audit.log
+          FILTER=gzip queue /tmp/artifacts/nodes/masters-kube-apiserver-audit.log.gz oc adm node-logs --role=master --path=kube-apiserver/
```
Comment (smarterclayton) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: This is just going to show you a directory listing unless you changed something in Kubelet...

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 2]
```diff
@@ -447,7 +447,18 @@ objects:
           done < /tmp/containers
 
           echo "Gathering kube-apiserver audit.log ..."
-          FILTER=gzip queue /tmp/artifacts/nodes/masters-kube-apiserver-audit.log.gz oc adm node-logs --role=master --path=kube-apiserver/audit.log
+          oc adm node-logs --role=master --path=kube-apiserver/ > /tmp/kube-audit-logs
+          while IFS=$'\n' read -r line; do
+            IFS=' ' read -ra log <<< "${line}"
+            FILTER=gzip queue /tmp/artifacts/nodes/"${log[0]}"-"${log[1]}".gz oc adm node-logs "${log[0]}" --path=kube-apiserver/"${log[1]}"
+          done < /tmp/kube-audit-logs
```
Comment (soltysh) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: @smarterclayton ptal now? the only downside of this approach is that we're getting each audit file in a separate file, not sure if the current `queue` implementation handles writing to it several times, but I haven't seen that

Comment (smarterclayton) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: I wouldn't try.  Probably would be too much work.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 3]
```diff
@@ -404,6 +404,8 @@ objects:
           queue /tmp/artifacts/secrets.json oc --insecure-skip-tls-verify --request-timeout=5s get secrets --all-namespaces -o json
           queue /tmp/artifacts/services.json oc --insecure-skip-tls-verify --request-timeout=5s get services --all-namespaces -o json
 
+          queue /tmp/artifacts/openapi.json oc --insecure-skip-tls-verify --request-timeout=5s get --raw /openapi/v2
```
Comment (smarterclayton) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: Please gzip this, it will be massive

Comment (soltysh) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: done

**Comment by smarterclayton** [comment]
/lgtm

hold on to your butts

**Pull request merged**